help/en/index.html

```diff
@@ -162,7 +169,7 @@ <h3>Navigation</h3>
       </ul>
     </div>
     <div class="footer" role="contentinfo">
-        &#169; Copyright 2021, Kartoza (Pty) Ltd..
+        &#169; Copyright 2013, Linfiniti Consulting CC..
```

Comment:
@vermeulendivan can you also update this the copyright text and the rest to the use the right text